### DISCUSSION TITLE
Will new configs be added?

### DISCUSSION THREAD

**Discussion by TKDev7** [spear-ai/citizen#198] [General]
Do you plan on expanding the configs and perhaps include markdown, vitest/playwright and stylelint?

**Comment by psirenny** [comment]
Yes, we plan to add more. Those are good suggestions.

**Markdownlint**
I think we faced a conflict between Prettier's markdown formatting and Markdownlint. But I'll have to revisit to make sure.

**Vitest**
We use vitest a lot, so it makes sense to add that as well.

**Stylelint**
Hadn't considered adding a config for this yet because our use of CSS is pretty minimal. Mostly it's a few Tailwind import statements followed by a couple lines of CSS. But we should consider adding this as well.

**Reply by TKDev7** [reply]
Perfect, thanks!
